platform/viewer/src/components/Header/Header.js

```diff
@@ -24,6 +26,10 @@ function Header(props) {
 
   const [options, setOptions] = useState([]);
 
+  const setHotkeys = hotkeys => {
```

Comment:
Somehow, when I pass the whole `hotkeysManage` instance into the modal `customProps`, the function lost the `hokeysManager` context (this)  and failed to set the hotkeys.  Any insight?

Reply:
I would avoid passing managers as props, managers are very app context-specific and should only be used inside its domain. If there's a need to pass the behaviour as a prop, use managers inside callbacks. This avoids modal to know too much. About losing this, you could explicitly bind before passing?